aerospike/datadog_checks/aerospike/data/conf.yaml.example

```diff
@@ -0,0 +1,377 @@
+init_config:
```

Comment:
Can we just put that at the end after `instances` ? So we have a quicker access to the actual configuration.

Reply:
Done. We'll move config to new format in future PR

Reply:
https://github.com/DataDog/integrations-core/pull/3082